public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -133,6 +134,18 @@ export function ExploreToolbar({ exploreId, topOfViewRef, onChangeTime }: Props)
       <PageToolbar
         aria-label="Explore toolbar"
         leftItems={[
+          config.featureToggles.exploreContentOutline && (
+            <ToolbarButton
+              key="content-outline"
+              variant="canvas"
+              tooltip="Content Outline"
```

Comment:
we generally use sentence case for the UI
```suggestion
              tooltip="Content outline"
```